src/components/layouts/collection-page-layout.tsx

```diff
@@ -506,13 +502,7 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                   </div>
                 </div>
 
-                <div className="flex flex-row space-x-3 text-sm opacity-80 md:items-start">
-                  {count > 1 && (
-                    <span>
-                      {count} learners are watching this course right now.
-                    </span>
-                  )}
-                </div>
+                <ActiveLearners courseSlug={course.slug} />
```

Comment:
put this in an external component because this layout is very large and difficult to navigate